src/components/color-picker/color-picker.e2e.ts

```diff
@@ -64,10 +64,6 @@ describe("calcite-color-picker", () => {
         propertyName: "allowEmpty",
         defaultValue: false
       },
-      {
-        propertyName: "appearance",
```

Comment:
This won't matter after my recommendation to keep assigning the defaults, but the `color-picker` was not included in the scope portion of the PR title.